test/rest-adapter.test.js

```diff
@@ -220,17 +220,17 @@ describe('RestAdapter', function() {
     describe('getHttpMethod', function() {
       it('returns POST for `all`', function() {
         var method = givenRestStaticMethod({ http: { verb: 'all' }});
-        expect(method.getHttpMethod()).to.equal('POST');
+        expect(method.getHttpMethod()[0]).to.equal('POST');
```

Comment:
With the ability to get all methods in place, we should test all members of the array returned.

```js
expect(method.getAllHttpMethods()).to.eql(['POST']);
```